my-app/src/App.js

```diff
@@ -0,0 +1,23 @@
+import Header from "./components/Header/Header";
+import Main from "./components/Main/Main";
+import Footer from "./components/Footer/Footer";
+import "./App.css";
+
+function App() {
+  const navList = ["Menu", "SearchBar", "Else"];
+
+  const items = navList.map((elem, i) => (
+    <li key={i} className="li">
```

Comment:
It's better not to use list of ```<li>``` tags without wrapping with ```<ul>``` tag to prevent future problems